tests/x11/gnomecase/nautilus_cut_file.pm

```diff
@@ -23,11 +23,15 @@ sub run {
     send_key "ctrl-x";
     send_key_until_needlematch 'nautilus-Downloads-matched', 'left', 5;
     send_key "ret";
-    send_key "ctrl-v";    #paste to dir ~/Downloads
-    assert_screen "nautilus-newfile-moved";
-    send_key "alt-up";                      #back to home dir from ~/Downloads
-    assert_screen 'nautilus-no-newfile';    #assure newfile moved
-    send_key "ctrl-w";                      #close nautilus
+    send_key "ctrl-v";
```

Comment:
I would vote for keeping comments here and below ( lines 28-30 ). Rare case when they really explains something